src/company/company.service.ts

```diff
@@ -13,7 +13,10 @@ export class CompanyService {
 
         
     ){}
-    
+    async getCompany(): Promise<Company[]>{
+        return await this.companyRepository.find({relations:["tags","recognitions","users"]});
```

Comment:
I think we should only return tags here, recs and users could be really long lists and would be better served in a separate endpoint where we can eventually add pagination to mitigate that issue